lib/cgrates_web_jsonapi_web/views/tp_route_view.ex

```diff
@@ -0,0 +1,25 @@
+defmodule CgratesWebJsonapiWeb.TpRouteView do
+  use CgratesWebJsonapiWeb, :view
+  use JaSerializer.PhoenixView
+
+  attributes([
+    :pk,
```

Comment:
there is no `:pk` field in your schema (tariff_plans/tp_route.ex). remove this line